padrino/manifest.yml

```diff
@@ -0,0 +1,3 @@
+---
+applications:
+-  command: bundle exec padrino start -p $PORT
```

Comment:
@dhcole - Lack of a name will error immediately. As I understand it, the manifest is parsed / validated entirely by the CLI which makes API calls based on the contents.